scipy/signal/_signaltools.py

```diff
@@ -1402,6 +1408,11 @@ def convolve(in1, in2, mode='full', method='auto'):
         result_type = np.result_type(volume, kernel)
         if result_type.kind in {'u', 'i'}:
             out = np.around(out)
+
+        if np.isnan(out.flat[0]) or np.isinf(out.flat[0]):
+            warnings.warn('Use of fft convolution on input with NAN or inf'
+                          'results in NAN or inf output.')
```

Comment:
This is missing a space and a stacklevel.

```suggestion
            warnings.warn('Use of fft convolution on input with NAN or inf'
                          ' results in NAN or inf output.',
                          stacklevel=2)
```

Reply:
Crazy that I still forget to comment on this sometimes!